README.rst

```diff
@@ -343,11 +343,29 @@ User Facing
 ~~~~~~~~~~~
 
 - Overhauled the code generation package to make the generators more easily
-  extensible and to improve simluation speed. [PR `#113`_]
+  extensible and to improve simulation speed. [PR `#113`_]
+- The visualizer has been overhauled as part of Tarun Gaba's 2014 GSoC
+  internship [PR `#82`_]. Here are some of the changes:
+
+  - The JavaScript is now handled by AJAX and requires a simple server.
+  - The JavaScript has been overhauled and now uses prototype.js for object
+    oriented design.
+  - The visualizer can now be loaded in an IPython notebook via IPython's
+    widgets.
+  - A slider was added to manually control the frame playback.
+  - The visualization objects can be manipulated via the GUI.
+  - The scene json file can be edited via the GUI.
+  - The data and scene description have been separated into two json files.
+  - Different simulation files can be loaded via the GUI.
+  - Shapes can now have a material.
+  - Simulation parameters can be modified and the simulations can be rerun all
+    via the GUI.
```

Comment:
Yes, it essentially works. But even so, all of these changes are merged into master already, so the release notes should reflect what is in master.

I'm starting to work on the viz bugs associated with these changes now. But I don't see any reason not to merge this pr.